src/libraries/Microsoft.PowerFx.Core/Functions/ISupportsArgumentSuggestions.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Collections.Generic;
+using Microsoft.PowerFx.Core.Types;
+using static Microsoft.PowerFx.Intellisense.ArgumentSuggestions;
+
+namespace Microsoft.PowerFx.Core.Functions
+{
+    // Used for Intellisense in connectors
+    internal interface ISupportsArgumentSuggestions
+    {
+        IEnumerable<KeyValuePair<string, DType>> GetArgumentSuggestions(TryGetEnumSymbol tryGetEnumSymbol, bool suggestUnqualifiedEnums, DType scopeType, int argumentIndex, out bool requiresSuggestionEscaping);
```

Comment:
>string [](http://example.com/codeflow?start=33&length=6)

can we document what this is? 

What exactly is string and Dtype here?

Reply:
Removed